docs/gaming/distribution.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Distribution
+head:
+  - - meta
+    - name: title
+      content: Solana Cookbook | Game Distribution
+  - - meta
+    - name: og:title
+      content: Solana Cookbook | Game Distribution
+  - - meta
+    - name: description
+      content: Different options on how to distribute your game built on Solana
+  - - meta
+    - name: og:description
+      content: Different options on how to distribute your game built on Solana
+  - - meta
+    - name: og:image
+      content: https://solanacookbook.com/cookbook-sharing-card.png
+  - - meta
+    - name: og:image:alt
+      content: Solana splash card
+  - - meta
+    - name: twitter:card
+      content: summary
+  - - meta
+    - name: twitter:site
+      content: "@solanacookbook"
+  - - meta
+    - name: twitter:image
+      content: "https://solanacookbook.com/cookbook-sharing-card.png"
+  - - meta
+    - name: robots
+      content: index,follow,noodp
+  - - meta
+    - name: googlebot
+      content: index,follow
+footer: MIT Licensed
+---
+
+# Distribution
+
+Distribution of your game depends highly on the platform you are using of course. With Solana game SDKs you can build for IOS, Android, Web and Native Windows or Mac. Using the Unity SDK you could even connect Nintendo Switch or XBox to Solana theoretically. Many game companies are pivoting to a mobile first approach because there are so many people with mobile phones in the world. Mobile comes with its own complications though, so you should pick what fits best to your game.  
```

Comment:
```suggestion
Distribution of your game depends highly on the platform you are using. With Solana, there are game SDKs you can build for IOS, Android, Web and Native Windows or Mac. Using the Unity SDK you could even connect Nintendo Switch or XBox to Solana theoretically. Many game companies are pivoting to a mobile first approach because there are so many people with mobile phones in the world. Mobile comes with its own complications though, so you should pick what fits best to your game.  
```